Makefile

```diff
@@ -46,24 +46,38 @@ run-simple-system: sw-simple-hello | $(Vibex_simple_system)
 		--raminit=$(simple-system-program)
 
 
-# Arty A7 100T
-# Use the following targets:
-# - "build-arty"
-# - "program-arty"
+# Arty A7 FPGA example
+# Use the following targets (depending on your hardware):
+# - "build-arty-35"
+# - "program-arty-35"
+# - "build-arty-100"
+# - "program-arty-100"
 arty-sw-program = examples/sw/led/led.vmem
 sw-led: $(arty-sw-program)
 
 .PHONY: $(arty-sw-program)
 $(arty-sw-program):
 	cd examples/sw/led && $(MAKE)
 
-build-arty: sw-led
+.PHONY: build-arty-35
+build-arty-35: sw-led
+	fusesoc --cores-root=. run --target=synth --setup --build \
+		lowrisc:ibex:top_artya7 --part xc7a35ticsg324-1L
+
+.PHONY: build-arty-100
+build-arty-100: sw-led
 	fusesoc --cores-root=. run --target=synth --setup --build \
-		lowrisc:ibex:top_artya7_100
+		lowrisc:ibex:top_artya7  --part xc7a100tcsg324-1
+
+.PHONY: program-arty-35
+program-arty-35:
+	fusesoc --cores-root=. run --target=synth --run \
+		lowrisc:ibex:top_artya7 --part xc7a35ticsg324-1L
 
-.PHONY: program-arty
-program-arty:
-	fusesoc --cores-root=. pgm lowrisc:ibex:top_artya7_100
+.PHONY: program-arty-100
+program-arty-100:
+	fusesoc --cores-root=. run --target=synth --run \
+		lowrisc:ibex:top_artya7 --part xc7a100tcsg324-1L
```

Comment:
"1L" might be a copy and paste error, but as with the above comment, I would just not specify it here.

Reply:
It is, but I wonder why they chose the low-power version of the smaller FPGA and the normal one for the bigger FPGA... doesn't make any sense from a technical standpoint :) Thanks for catching that!